src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlConnectionStringBuilderTest.cs

```diff
@@ -37,7 +37,7 @@ public class SqlConnectionStringBuilderTest
         [InlineData("Enclave Attestation Url = http://dymmyurl")]
         [InlineData("Encrypt = true")]
         [InlineData("Enlist = false")]
-        [InlineData("Failover Partner = randomserver.sys.local")]
+        [InlineData("Initial Catalog = Northwind; Failover Partner = randomserver.sys.local")]
```

Comment:
Why change this?

Reply:
Maybe keep both tests?

Reply:
Required for creating instance of `SqlConnection`

Reply:
> Maybe keep both tests?

Since the goal of the test is to validate connection property, and it can only exist/be accepted by driver if it's alongside "Initial Catalog", adding `Initial Catalog` here doesn't hurt to this testcase.